.github/workflows/go_ci.yaml

```
@@ -0,0 +1,38 @@
+ build:
+    name: Build and Test (Go)
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+    services:
+      dind:
+        image: docker:23.0-rc-dind-rootless
+        ports:
+          - 2375:2375
+    steps:
+    - name: Checkout PR branch
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)